seatunnel-config/pom.xml

```diff
@@ -35,6 +35,96 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <skip.pmd.check>true</skip.pmd.check>
+        <seatunnel.shade.package>org.apache.seatunnel.shade</seatunnel.shade.package>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.typesafe</groupId>
+            <artifactId>config</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+
+        <finalName>${project.artifactId}-${project.version}</finalName>
+
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.2.4</version>
```

Comment:
Maybe we should use `${maven-shade-plugin.version}` instead, which is `2.4.3`, or just delete this line, and it will be the same as the version in `pluginManagement` in parent module.

Reply:
You are right, thanks.

Reply:
You are welcome